src/BaseServiceProvider.php

```diff
@@ -111,6 +113,11 @@ public function registerAdminMiddleware(Router $router)
         }
     }
 
+    public function registerCustomAuthGuard()
+    {
+        View::composer('backpack::*', \Backpack\Base\app\Http\ViewComposers\AuthComposer::class);
```

Comment:
I'm ok with having both a facade and a helper function. Just like the `auth` helper in Laravel. The documentation says: "The auth function returns an authenticator instance. You may use it instead of the Auth facade for convenience:". I'll update the code if you're ok with that :)